dts/bindings/usbc/vbus-voltage-divider-adc.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: USBC ADC Measurement and control
+
+compatible: "vbus-voltage-divider-adc"
+
+include: [base.yaml, voltage-divider.yaml]
+
+properties:
+    discharge-gpios:
+       type: phandle-array
+       required: false
```

Comment:
where is this used? can't see any usage in https://github.com/zephyrproject-rtos/zephyr/commit/70f420f53a8d92c0ce80cb6ecaef5c8aba966632

Reply:
It's used here [b5dfa79](https://github.com/zephyrproject-rtos/zephyr/pull/45601/commits/b5dfa799b88dc87965ccb79faba655710688cc1c)

		/* Pin B14 is used to control VBUS Discharge for Port2 */
		discharge-gpios = <&gpiob 14 GPIO_ACTIVE_HIGH>;